system/addon-manifest.xml

```diff
@@ -29,6 +29,7 @@
   <addon>screensaver.xbmc.builtin.black</addon>
   <addon>screensaver.xbmc.builtin.dim</addon>
   <addon>script.module.pil</addon>
+  <addon>script.module.pycryptodome</addon>
```

Comment:
yes, that's exactly what it is doing (and you can not disable/unintall those addons), @ronie